common/beerocks/tlvf/yaml/beerocks/tlvf/beerocks_message_bml.yaml

```
@@ -217,7 +217,22 @@ cACTION_BML_CHANGE_MODULE_LOGGING_LEVEL_RESPONSE:
 
 cACTION_BML_WIFI_CREDENTIALS_SET_REQUEST:
   _type: class
-  params: sWifiCredentials
+  al_mac: sMacAddr
+  ssid:
+   _type: char 
+   _length: [ "beerocks::message::WIFI_SSID_MAX_LENGTH" ]
```

(nitpick) you could have used a dynamic length here, like for operating channels.:
```suggestion
  ssid_length:
    _type: uint8_t
    _length_var: True
  ssid:
   _type: char 
   _length: [ ssid_length ]
```